File: 준영/LV2/피보나치 수.js
function solution(n) {
  var answer = 0
  var x = 0
  var y = 1
  for (let i = 0; i < n-1; i++) {
    var temp = x + y
    x = y % 1234567
    y = temp % 1234567
  }
  answer = y
  return answer
}

File: 준영/프로그래머스/LV2/피보나치 수.py
def solution(n):
    answer = 0
    fibo = [0, 1]
    for i in range(n-1):
        fibo.append(fibo[-1] + fibo[-2])
    answer = fibo[-1] % 1234567
    return answer

solution(3)
solution(100)